test/token-paymaster/biconomy-token-paymaster-specs.ts

```diff
@@ -544,6 +544,69 @@ describe("Biconomy Token Paymaster", function () {
       ).to.be.reverted;
     });
 
+    it("should revert if price markup charged is too darn high", async ()  => {
+
+      const userSCW: any = BiconomyAccountImplementation__factory.connect(walletAddress, deployer)
+
+      await token
+        .connect(deployer)
+        .transfer(walletAddress, ethers.utils.parseEther("100"));
+
+      // We make transferFrom impossible by setting allowance to zero
+      const userOp1 = await fillAndSign(
+        {
+          sender: walletAddress,
+          verificationGasLimit: 200000,
+          // initCode: hexConcat([walletFactory.address, deploymentData]),
+          // nonce: 0,
+          callData: encodeERC20Approval(
+            userSCW,
+            token,
+            paymasterAddress,
+            ethers.constants.Zero 
+          ),
+        },
+        walletOwner,
+        entryPoint,
+        "nonce"
+      );
+
+      console.log("userOp");
```

Comment:
Remove console.log. It should be used only during development.

Reply:
ok